modules/setting/setting.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	"github.com/strk/go-libravatar"
```

Comment:
For Gogs, that's not, the repository just gives general purpose idea, which disallow local path import.